ui/raidboss/data/03-hw/raid/a7s.txt

```diff
@@ -235,15 +235,15 @@ hideall "--sync--"
 1326 "Bombs x2" Ability { id: "15EE", source: "Quickthinx Allthoughts" } window 40,10 jump 1526
 1333 "Jails" #Green Tether, Purple Prey
 1335 "Small Doll"
-1336 "Zoomdoom" Ability { id: "15BE", source: "Quickthinx Allthoughts" } window 55
+1336 "Zoomdoom" Ability { id: "15BE", source: "Quickthinx Allthoughts" } window 27.5,27.5
```

Comment:
I understand the basic logic of `55/2=27.5` but I think for readability/consistency it makes more sense to make these syncs `30/30` instead, unless you can see some reason they need to be exactly 27.5?

Feel free to disagree here, as well. This is really just a nitpick.

Reply:
> I understand the basic logic of `55/2=27.5` but I think for readability/consistency it makes more sense to make these syncs `30/30` instead, unless you can see some reason they need to be exactly 27.5?

This was me trying to fix lint only and avoid any substantive change that would affect timeline behavior (unless it was an obvious mistake).  I didn't want to adjust any windows without knowing for certain it wouldn't impact how the timeline runs.  That said, given the age of this fight and what I've now seen in other Alex timelines, it's better than even money that this window was probably intended to be 55,55 (or more likely, 55,0).

Given how old the content is and the fact that these timelines probably have many other issues, I'm fine with making these two syncs `30,30` - and if we get a bug report, we can revisit.